server/helpers/responseMessage.js

```diff
@@ -1,6 +1,6 @@
-export default (response, statusCode, object) => {
+export default (response, statusCode, object, status = 'failure') => {
   response.status(statusCode).json({
-    status: 'failure',
```

Comment:
Revert the changes in this function. It shouldn't even show up in your changed files.

Reply:
I was asked to use the responseMessage function. The only way to show success is either to create a new function which will lead to duplication or to add that.

Reply:
Please check my latest comment above

Reply:
Done